src/main/resources/META-INF/mods.toml

```diff
@@ -3,7 +3,6 @@ loaderVersion="[24,]"
 issueTrackerURL="http://www.minecraftforge.net/"
 logoFile="forge_logo.png"
 license="LGPL v2.1"
-
```

Comment:
You only changed this file with whitespace.

Reply:
I previously had fishing_example in this toml. When I moved it to src/test, there appears to be a leftover whitespace. Is there a way to completely remove the mods.toml from the PR here?

Reply:
The changes to this file still need to be discarded.

Reply:
I ended up editting the file in hex to make it the same as the one from the main github. If it isn't good now, it isn't going to be.